metabench.cmake

```diff
@@ -274,11 +274,11 @@ file(WRITE ${METABENCH_RB_PATH}
 ################################################################################
 set(MEASURE_RB_PATH ${METABENCH_DIR}/measure.rb)
 file(WRITE ${MEASURE_RB_PATH}
-"require 'benchmark'                                                    \n"
-"command = ARGV.join(' ')                                               \n"
-"time = Benchmark.realtime { `#{command}` }                             \n"
-"puts \"[command line: #{command}]\"                                    \n"
-"puts \"[compilation time: #{time}]\"                                   \n"
+"require 'benchmark'                                                         \n"
+"command = ARGV.map{ |arg| arg.match(/\\s/) ? '\"'+arg+'\"' : arg }.join(' ')\n"
```

Comment:
I think there's a different issue here, probably an issue with `ctest`, because otherwise we wouldn't have any report from `cl` on STDERR, but we do

> cl : Command line error D8003 : missing source filename